app/src/main/java/com/example/keniphhomemaintenance/dwellings/DwellingType.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.keniphhomemaintenance.dwellings
+
+enum class DwellingType(val type: String) {
```

Comment:
Thought it would be helpful when creating default utilities/repair items (`DwellingType.House` could have water heater checked by default, whereas `DwellingType.Car` would not)

Reply:
Don't disagree, but is a Car or a Boat a Dwelling?

Reply:
nah but we talked about adding these at some point, this would entail renaming "Dwelling" to something more appropriately scoped but this is all deleted for now anyway